### PR TITLE
Adds k'ois contaminated phoron reagent

### PR DESCRIPTION
**Pull request by Sniblet**
Wow! My very first PR here and it's kinda big! I'm a bad coder and phoron is important so please review a lot!!

**This splits reagent-form phoron into four types:**
**toxin/phoron**, which is now abstract and does not appear ingame;
**toxin/phoron/base**, which does all the normal unobtanium things that phoron does and is what you get from grinding it/injecting with a vaurca hardsuit/eating filtered k'ois bars/etc;
**toxin/phoron/kois**, which is only good as phoron for vaurca consumption (no chemical reactions except sedantian firestorm, doesn't work for sabotaging light tubes or whatever, etc), works as a secondary mycosis vector, and spawns in unclean k'ois snacks, k'ois mycosis procs, and the plant itself;
**toxin/phoron/kois/black**, which spawns in black k'ois and does black k'ois things.

All of these will show as "Phoron" to most chemical machines, but if analyzed by a chemmaster, k'ois variants will have a bonus description sentence about biological traces. Don't drink phoron! You can never be sure that it's safe- wait.

This doesn't add gaseous k'ois phoron, partly because oh my god that's a lot of work and I'm GOING to break something trying that, partly because it'd need separate art that looks less energetic, and partly because people who're playing with k'ois phoron aren't really talking to atmos that much. If you spill phoron/kois into the air you get "safe" and functional phoron gas.
Look.
This is an improvement.
I mean, if it works.
😩

### CODE REVIEW

**Comment by Sniblet** [comment]
!review required

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm [hunk 1]
```diff
@@ -156,6 +156,58 @@
...
+			var/obj/item/organ/internal/parasite/P = M.internal_organs_by_name["blackkois"]
+		if(istype(P) && P.stage >= 3)
+			is_vaurcalike = TRUE
+		if(is_vaurcalike)
+			M.heal_organ_damage(1.4 * removed, 1.6 * removed)
+			M.adjustNutritionLoss(-nutriment_factor * removed)
+		else
+			infect(M, alien, removed)
+
+	/decl/reagent/toxin/phoron/kois/affect_blood(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
```
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm:182]: This and all the procs below are indented one to the right too much.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm (hunk 1 above)
Comment (Sniblet) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm]: I did do that, I think I fixed it

**Comment by Sniblet** [comment]
I have been told how to changelog and I did it

**Comment by Sniblet** [comment]
god what a mess this can't be how im supposed to do it

**Comment by Sniblet** [comment]
It turns out that while trying to solve merge conflicts I have accidentally reduced all of my work to atoms
I'll need a fresh branch and 3 hours

**Pull request closed without merging**